tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -375,6 +375,49 @@ Search Items In Resources Section
     Sleep   5
     Input Text  xpath://input[@class="pf-c-search-input__text-input"]       ${element}
 
+Open Element By Name
+    [Documentation]     It opens the element by enabling Quick Start filter in Resources Section of ODS Dashboard.
+    [Arguments]     ${element}
+    Click Link      Resources
+    Sleep   5
+    Select Checkbox     //input[@id="quickstart--check-box"]
+    Wait Until Page Contains Element    //article[@id="${element}"]
+    Click Link          //article[@id="${element}"]//a
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+
+Get The Count Of Description
+    [Documentation]     Gives the count of description which is shown while starting the tour.
+    ${count}=   Get ELement Count   //div[@class="pfext-quick-start-content"]/ul/li
+    Click Button    //button[@data-test="Start button"]
+    Wait Until Page Contains Element    //div[@class="pfext-quick-start-content"]
+    [Return]    ${count}
```

Comment:
not sure about the scope of the this kw. It seems to get some element count, but then it clicks on a Start button, which IMO seems to be a separate action/kw

Reply:
I am using get element count so that I  know that I have to move to the next desciption for how many times. Instead of writing for 3-4 times for all the cases I have binded them in a keyword. And regarding the start button we can change the name of keyword so that it makes sense. will it be a good option?

Reply:
hm I mean, the kw objective is to get the number of steps in the quick start? If yes, then you can click the "start" button after you call this kw (for example, line 186)

Reply:
Got it. And I have made the changes.